locale/en/submission.po

```diff
@@ -252,7 +252,7 @@ msgid "publication.datePublished"
 msgstr "Date Published"
 
 msgid "publication.editDisabled"
```

Comment:
This locale key is also used for presentation to the author, where editing will still be disabled in some cases, so this key should be left as-is and a new locale key created for the editor case.